helm/vertical-pod-autoscaler-app/templates/admission-controller-monitoring-service.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "vpa.fullname" . }}-admission-controller-monitoring
```

Comment:
Why do we need a service specifically for monitoring?

Reply:
Because our current scrape config for managed apps is using service labels and not pod labels as some apps have the annotations on the service

Reply:
We could actually allow both pod and service labels but this scrape config should be difficult to read and maintain

Reply:
I've got no issue with using Services for scraping, my thinking was more that if we need to add them for scraping, why not make them generic, or use the generic ones - i.e: use `{{ include "vpa.fullname" . }}-admission-controller `

Reply:
Well, the admission controller is the only deployment with a service. I picked the "add another service" approach used in nginx-ingress-controller-app to be able to not have the annotations on the other service but we can use only one. @team-atlas wdyt?

Reply:
But, could we not add one service for all pods here? It seems  the targets are the same, no?

Reply:
This PR adds a service per components as the pod labels have an extra `app.kubernetes.io/component` label. 
We have: 
- vpa-admission-controller-monitoring for the admission controller
- vpa-recommender-monitoring for the recommender
- vpa-updater-monitoring for the updater 

What we would like to decide on is if we use only one service for the admission controller or if we split it between 2 services, one for the webhook to call it, and one for the monitoring or if we merge the services into one for the admission controller only

Reply:
I tend to prefer adding monitoring into the already existing service.

Reply:
I merged the duplicate services for the admission controller

Reply:
ping @giantswarm/team-atlas